Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -90,6 +90,19 @@
   loadouts:
   - NoPro
 
+- type: loadoutGroup
+  id: LawyerStamps
+  name: loadout-group-lawyer-stamps
+  loadouts:
+  - RSLawyer
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)